templates/stamp/stamp-keyvault-v3.json

```diff
@@ -0,0 +1,1112 @@
+            "type": "securestring"
+        },
+        "mysqlBackupUserPassword": {
+            "type": "securestring"
+        },
+
+        "deploymentSlot": {
+            "type": "string",
+            "defaultValue": "slot1",
+            "allowedValues": [
+                "slot1",
+                "slot2"
+            ],
+            "metadata": {
+                "description": "Deployment slot for the frontend"
+            }
+        },
+        "autoScaleEnabled": {
+            "type": "bool",
+            "defaultValue": "false",
+            "metadata": {
+                "description": "Indicator of whether or not autoscaling would be enabled on the Vm Scale Set"
+            }
+        },
+        "autoScaleCapacityDefault": {
+            "type": "int",
+            "maxValue": 20,
+            "defaultValue": 5,
+            "metadata": {
+                "description": "Default number of application virtual machines within the VM Scale Set."
+            }
+        },
+        "autoScaleCapacityMaximum": {
+            "type": "int",
+            "maxValue": 20,
+            "defaultValue": 20,
+            "metadata": {
+                "description": "Maximum number of application virtual machines within the VM Scale Set."
+            }
+        },
+        "autoScaleCapacityMinimum": {
+            "type": "int",
+            "maxValue": 20,
+            "defaultValue": 1,
+            "metadata": {
+                "description": "Default number of application virtual machines within the VM Scale Set."
+            }
+        },
+        "oxaToolsGithubAccountName": {
+            "type": "string",
+            "defaultValue": "microsoft",
+            "metadata": {
+                "description": "Name of the account that owns the oxa-tools GitHub repository"
+            }
+        },
+        "oxaToolsGithubProjectName": {
+            "type": "string",
+            "defaultValue": "oxa-tools",
+            "metadata": {
+                "description": "Name of the oxa-tools GitHub repository"
+            }
+        },
+        "oxaToolsGithubBranch": {
+            "type": "string",
+            "defaultValue": "master",
+            "metadata": {
+                "description": "Branch of the oxa-tools GitHub repository"
+            }
+        },
+        "edxConfigurationGithubAccountName": {
+            "type": "string",
+            "defaultValue": "microsoft",
+            "metadata": {
+                "description": "Name of the account that owns the edx configuration repository"
+            }
+        },
+        "edxConfigurationGithubProjectName": {
+            "type": "string",
+            "defaultValue": "edx-configuration",
+            "metadata": {
+                "description": "Name of the edx configuration GitHub repository"
+            }
+        },
+        "edxConfigurationGithubBranch": {
+            "type": "string",
+            "defaultValue": "oxa/master",
```

Comment:
shouldn't this be oxa/master.fic ?

Reply:
Not necessarily. For now, yes oxa/master.fic seems to be the correct value. When we upgrade, it will again become invalid and we begin chasing these updates and add additional change overhead. As the system is configured, this value doesn't matter since it is will always be overridden and we therefore don't need to continually make such updates.

Reply:
then we should change the string to something static that doesn't look like a valid branch. my strong recommendation: we either
a) initialize default values to something completely meaningless (like an empty string) or
b) the most likely final valuer (oxa/master.fic) which can easily be "grep'd" for when we need to update it -- (my original suggestion)

Reply:
Like I said, it isn't necessary. It may be good for aesthetics though which we can come around to in a later iteration.